changelog.txt

```diff
@@ -23,6 +23,7 @@ that repo.
 - `suspendmanager`: does not suspend non blocking jobs such as floor bars or bridges anymore
 - `suspendmanager`: fix occasional bad identification of buildingplan jobs
 - `warn-starving`: no longer warns for enemy and neutral units
+- `gui/gm-editor`: no longer nudges last open window when opening a new one
```

Comment:
this file looks out of date -- after yesterday's release, these lines are now in the "50.07-beta2" section. Could you merge current HEAD or rebase, then move these new lines to the new "Future" section?

Reply:
Whoops! - will do